images/server/Dockerfile.fedora

```diff
@@ -1,13 +1,13 @@
 FROM quay.io/samba.org/sambacc:latest AS builder
-ARG SAMBACC_VER=f6480c5861a56ef9d1ebb965aebb14732ec2690c
+ARG SAMBACC_VER=fd747527ed86d01821c4a3b72f1c4f893bb1ca36
```

Comment:
With https://github.com/samba-in-kubernetes/sambacc/pull/26 merged, are we still gonna stay with commit hash and not `master` branch?

Reply:
Right now, I would prefer to stick with the hash. It helps me know exactly what we are taking into the container. I'm not against switching it, but I'd like to have something in place wrt samba-in-kubernetes/sambacc/issues/20  first